Microsoft.Toolkit.Uwp.UI.Media/Microsoft.Toolkit.Uwp.UI.Media.csproj

```diff
@@ -43,4 +43,8 @@
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
     <PackageReference Include="Win2D.uwp" Version="1.25.0" />
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.Toolkit\Microsoft.Toolkit.csproj" />
```

Comment:
@ratishphilip you removed the bits that you had originally put here, eh? So we should remove the reference from here as well.

Reply:
I am still using the ThrowHelper in the Parser code. So this reference should remain.